x-pack/plugins/serverless_search/public/layout/nav.tsx

```diff
@@ -12,20 +12,114 @@ import {
   NavigationKibanaProvider,
 } from '@kbn/shared-ux-chrome-navigation';
 import React from 'react';
+import { i18n } from '@kbn/i18n';
+
+const NAVIGATION_PLATFORM_CONFIG = {
+  analytics: { enabled: false },
+  ml: { enabled: false },
+  devTools: { enabled: false },
+  management: { enabled: false },
+};
 
 const navItems: ChromeNavigationNodeViewModel[] = [
   {
-    title: '',
-    id: 'root',
+    id: 'search_getting_started',
+    title: i18n.translate('xpack.serverlessSearch.nav.gettingStarted', {
+      defaultMessage: 'Getting started',
+    }),
+    href: '/app/elasticsearch',
+  },
+  {
+    id: 'dev_tools',
+    title: i18n.translate('xpack.serverlessSearch.nav.devTools', { defaultMessage: 'Dev Tools' }),
+    items: [
+      {
+        id: 'dev_tools_console',
+        title: i18n.translate('xpack.serverlessSearch.nav.devTools.console', {
+          defaultMessage: 'Console',
+        }),
+        href: '/app/dev_tools#/console',
```

Comment:
That `#` looks odd on these but I assume it is expected.

Reply:
yea, it ensures the correct tab is selected on the dev_tools page. I would expect us to update this soon to have an option to either not have tabs on that page or to have individual pages for the dev tools.